concurrency/1.1/apidocs/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0

```

Comment:
The review [checkist](https://github.com/jakartaee/specification-committee/blob/master/spec_review_checklist.md) says:  "no META-INF directory in PR"